composer.json

```
@@ -14,7 +14,7 @@
         }
     },
     "require": {
-        "php": "^7.2",
+        "php": ">=7.2",
```

this is bad practice. as this would also mean, that this is compatible with EVERY PHP version greater than 7.2 which includes PHP8 and all upcoming versions. 

if PHP8 is supported you should do the following:
```suggestion
        "php": "^7.2 || ^8.0",
```

---

ups, sorry, forgot to hit the draft button